### PR TITLE
Added Priestley Taylor method on Evapotranspiration dialog

### PR DESCRIPTION
**Pull request by MeSophie**
Fixes partially #7060 
@rdstern I added Priestley Taylor method on Evapotranspiration dialog. On this method I didn't remove the parameter **Missing method** I dont have more information about this parameter so I just added it for Priestley Taylor also.
By the way I notice that solar can be also **precipitation** (https://rdrr.io/cran/Evapotranspiration/man/ET.PriestleyTaylor.html) is there any reason that we didn't add precipitation on solar?
I also removed the copy of all the results in the output window.

@N-thony Please can you have a look.

### CODE REVIEW

**Comment by N-thony** [comment]
@derekagorhom can you peer review this?

File: instat/dlgEvapotranspiration.vb [hunk 1]
```diff
@@ -121,6 +120,13 @@ Public Class dlgEvapotranspiration
         ucrChkWind.SetValuesCheckedAndUnchecked(Chr(34) & "yes" & Chr(34), Chr(34) & "no" & Chr(34))
         ucrChkWind.SetRDefault(Chr(34) & "yes" & Chr(34))
 
+        ucrNudAlpha.SetParameter(New RParameter("alpha", 4))
+        ucrNudAlpha.SetMinMax(0, 1)
+        ucrNudAlpha.DecimalPlaces = 2
+        ucrNudAlpha.Increment = 0.05
+        'ucrNudAlpha.SetRDefault(0.23)
```
Comment (derekagorhom) [inline instat/dlgEvapotranspiration.vb:127]: If it is not needed can it be removed

File: instat/dlgEvapotranspiration.vb [hunk 2]
```diff
@@ -256,6 +278,16 @@ Public Class dlgEvapotranspiration
         clsHargreavesSamani.AddParameter("save.csv", Chr(34) & "no" & Chr(34), iPosition:=4)
         clsHargreavesSamani.SetAssignTo("Hargreaves_Samani")
 
+        clsETPriestleyTaylor.SetPackageName("Evapotranspiration")
+        clsETPriestleyTaylor.SetRCommand("ET.PriestleyTaylor")
+        clsETPriestleyTaylor.AddParameter("data", clsRFunctionParameter:=clsReadInputs, iPosition:=0)
+        clsETPriestleyTaylor.AddParameter("constants", "constants", iPosition:=1, bIncludeArgumentName:=False)
+        clsETPriestleyTaylor.AddParameter("ts", Chr(34) & "daily" & Chr(34), iPosition:=2)
+        clsETPriestleyTaylor.AddParameter("message", Chr(34) & "yes" & Chr(34), iPosition:=3)
+        ' clsETPriestleyTaylor.AddParameter("alpha", 0.23, iPosition:=4)
```
Comment (derekagorhom) [inline instat/dlgEvapotranspiration.vb:287]: Same with this line also

**Review by derekagorhom — COMMENTED** [review]
@MeSophie can you check on these reviews

File: instat/dlgEvapotranspiration.vb [hunk 3]
```diff
@@ -133,18 +139,21 @@ Public Class dlgEvapotranspiration
...
 
         ucrPnlMethod.AddToLinkedControls(ucrInputCrop, {rdoPenmanMonteith}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True)
-        ucrPnlMethod.AddToLinkedControls(ucrReceiverHumidityMax, {rdoPenmanMonteith}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True, bNewLinkedChangeToDefaultState:=True)
-        ucrPnlMethod.AddToLinkedControls(ucrReceiverHumidityMin, {rdoPenmanMonteith}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True, bNewLinkedChangeToDefaultState:=True)
-        ucrPnlMethod.AddToLinkedControls(ucrInputTimeStep, {rdoPenmanMonteith, rdoHargreavesSamani}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True)
+        ucrPnlMethod.AddToLinkedControls(ucrReceiverHumidityMax, {rdoPenmanMonteith, rdoPriestleyTaylor}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True, bNewLinkedChangeToDefaultState:=True)
+        ucrPnlMethod.AddToLinkedControls(ucrReceiverHumidityMin, {rdoPenmanMonteith, rdoPriestleyTaylor}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True, bNewLinkedChangeToDefaultState:=True)
```
Comment (derekagorhom) [inline instat/dlgEvapotranspiration.vb:149]: can you change this into 
`ucrPnlMethod.AddToLinkedControls({ucrReceiverHumidityMax, ucrReceiverHumidityMin}, {rdoPenmanMonteith, rdoPriestleyTaylor}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True, bNewLinkedChangeToDefaultState:=True)`

**Comment by MeSophie** [comment]
@derekagorhom Thank you. 
Please before review it again can you wait a bit? I'm still working on this dialog. 
I will let you know when I'm done. Thank you

**Comment by N-thony** [comment]
> @derekagorhom Thank you. Please before review it again can you wait a bit? I'm still working on this dialog. I will let you know when I'm done. Thank you

@MeSophie are you still making the changes?

**Comment by rdstern** [comment]
For information and planning I show the dialogues in the old Instat and these worked fine.  You first have site information - which we don't have yet:

image

And here is the PM dialogue:

image

That's 30 years old and we should be able to at least match this software!

**Comment by rdstern** [comment]
[climatic.pdf](https://github.com/africanmathsinitiative/R-Instat/files/10394222/climatic.pdf)

I have uploaded the old climatic guide.  Chapter 9 includes Penman Monteith.

**Comment by MeSophie** [comment]
These modifications has been done in PR #8118 and this branch has some problem so I will close this PR

**Pull request closed without merging**